llvm/docs/CodeReview.rst

```diff
@@ -103,6 +103,12 @@ ready to be committed. Specifically, once a patch is sent out for review, it
 needs an explicit approval before it is committed. Do not assume silent
 approval, or solicit objections to a patch with a deadline.
 
+.. note::
+   If you are using a Pull Request to get precommit CI results, but not
+   because it needs a precommit review, apply the `skip-precommit-approval
+   <https://github.com/llvm/llvm-project/labels?q=skip-precommit-approval>`
+   label to the PR.
```

Comment:
```suggestion
   If you are using a Pull Request to get precommit CI results or for a revert, but
   not because it needs a precommit review, apply the `skip-precommit-approval
   <https://github.com/llvm/llvm-project/labels?q=skip-precommit-approval>`
   label to the PR.
```

(adding revert here)

Reply:
Oh, yeah - generalized the wording a bit further to mention "purposes other than code review" including code review, convenient web-based reverts, "etc" since there might be other reasons now or in the future that PRs might be used for purposes other than review.